e2e/tests/adminManagesOrders_test.js

```diff
@@ -338,6 +338,37 @@ Scenario('Create C47A appointment of a Children\'s Guardian', async ({ I, caseVi
   });
 });
 
+Scenario('Create C43a special guardianship order', async ({I, caseViewPage, manageOrdersEventPage}) => {
+  await caseViewPage.goToNewActions(config.administrationActions.manageOrders);
+
+  await manageOrdersEventPage.selectOperation(manageOrdersEventPage.operations.options.create);
+  await I.goToNextPage();
+  await manageOrdersEventPage.selectOrder(manageOrdersEventPage.orders.options.c43a);
+  await I.goToNextPage();
+  manageOrdersEventPage.selectRelatedToHearing(manageOrdersEventPage.hearingDetails.linkedToHearing.options.no);
+  await I.goToNextPage();
+  manageOrdersEventPage.enterJudge();
+  await manageOrdersEventPage.enterApprovalDateTime(today);
+  await I.goToNextPage();
+  manageOrdersEventPage.selectOrderByConsent()
+  await manageOrdersEventPage.selectGuardian([0])
+  await manageOrdersEventPage.enterFurtherDirections('Further special guardianship details.');
+  await manageOrdersEventPage.selectIsFinalOrder();
+  await I.goToNextPage();
+  await manageOrdersEventPage.checkPreview();
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.administrationActions.manageOrders);
+  assertOrder(I, caseViewPage, {
+    orderIndex: 1,
```

Comment:
for future/tech debt: we should find better way to do this indexing, whole scenario must be ran together and it is not obvious what index is until you run suite and spot it yourself. Maybe we can locate element based on order type and grab index using that?

Reply:
I think it will come when we start to split the scenarios. It's becoming too long running at the moment